src/config/colors.toml

```diff
@@ -20,11 +20,7 @@
 "divergent change_id"="red"
 "conflict" = "red"
 
-# TODO: This near-duplication of the lines above is unfortunate. Should we
-# allow adding and clearing the "bright" bit somehow? Or should we instead
-# use a different background color? (We don't have support for background
```

Comment:
Perhaps this TODO should change to making it possible to set "bright" an attribute like "bold" and "underlined"? You could have a separate "bright_bg" trait, but it would probably be better to simply let people use "bright" both as an attribute or as a color name.

Or do you think it's not worth the trouble?

Reply:
Since "bright" only applies to the 8 base colors, I don't think we should do it. It would make it complicated when we add support for more than the 8 base colors.

Reply:
Good point. I'm not sure if `jj` should ever use more than 16 colors but we shouldn't restrict ourselves now.